sdk/cosmos/azure-cosmos/azure/cosmos/_execution_context/multi_execution_aggregator.py

```diff
@@ -129,6 +136,32 @@ def fetch_next_block(self):
 
         raise NotImplementedError("You should use pipeline's fetch_next_block.")
 
+    def _repair_document_producer(self):
```

Comment:
Would be good to add some documentation to this function, like what it does and how it repairs the document producers.